articles/azure-functions/functions-host-json.md

```diff
@@ -112,15 +116,16 @@ Controls the [sampling feature in Application Insights](./functions-monitoring.m
 
 |Property  |Default | Description |
 |---------|---------|---------| 
-|isEnabled|true|Enables or disables sampling.| 
-|maxTelemetryItemsPerSecond|20|The threshold at which sampling begins.| 
-|EnableLiveMetrics |true|Enables live metrics collection.|
-|EnableDependencyTracking|true|Enables dependency tracking.|
-|EnablePerformanceCountersCollection|true|Enables Kudu performance counters collection.|
+|enableDependencyTracking|true|Enables dependency tracking.|
+|enablePerformanceCountersCollection|true|Enables Kudu performance counters collection.|
+|samplingExcludedTypes|null|A semi-colon delimited list of types that you do not want to be sampled. Recognized types are: Dependency, Event, Exception, PageView, Request, Trace. All instances of the specified types are transmitted; the types that are not specified are sampled.| 
+|samplingIncludedTypes|null|A semi-colon delimited list of types that you want to be sampled. Recognized types are: Dependency, Event, Exception, PageView, Request, Trace. The specified types are sampled; all instances of the other types will always be transmitted.|
+|samplingSettings.isEnabled|true|Enables or disables sampling.| 
+|samplingSettings.maxTelemetryItemsPerSecond|20|The threshold at which sampling begins.|
 
 ## cosmosDb
 
-Configuration setting can be found in [Cosmos DB triggers and bindings](functions-bindings-cosmosdb-v2.md#host-json).
+Configuration setting can be found in [Cosmos DB triggers and bindings](functions-bindings-cosmosdb-v2.md#host-json).A semi-colon delimited list of types that you want to be sampled. Recognized types are: Dependency, Event, Exception, PageView, Request, Trace. The specified types are sampled; all instances of the other types will always be transmitted.
```

Comment:
Missing preceding space before added text. :)

Reply:
Actually that text shouldn't even be there -- good catch. I must have `ctrl+v`'ed a little too aggressively!

Reply:
Glad I could help :)